tests/ExtendedPdoTest.php

```diff
@@ -651,4 +651,18 @@ public function testWithProfileLogging()
         $this->pdo->query("SELECT 5 FROM pdotest");
         $this->assertEquals(4, count($logger->getLog()));
     }
+
+    public function testNoExportOfLoginCredentials()
+    {
+        $pdo = new ExtendedPdo('sqlite::memory:', 'username', 'password');
+        $debugInfo = $pdo->__debugInfo();
```

Comment:
Accoding to docs (see http://php.net/manual/en/language.oop5.magic.php#object.debuginfo) the `__debugInfo` method is called internally by `var_dump`. I think it would be more close to real-life scenario to call `var_dump($pdo)` and assert that printed fragments don't contain sensitive info.